aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/ModelMetadata.java

```diff
@@ -37,7 +38,7 @@
     private final @ModelField(targetType = "ID", isRequired = true) String id;
     private final @ModelField(targetType = "Boolean") Boolean _deleted;
     private final @ModelField(targetType = "Int") Integer _version;
-    private final @ModelField(targetType = "AWSTimestamp") Long _lastChangedAt;
+    private final @ModelField(targetType = "AWSTimestamp") Temporal.Timestamp _lastChangedAt;
```

Comment:
Just making sure: this change is scoped only to model metadata right? Codegen won't be affected?

Reply:
Correct, no impact on codegen.